ros/src/common/libs/diagnostics_lib/diag_lib/package.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0"?>
```

Comment:
Please remove unnecessary comment.
Also, please fill in all with reference to the following.
https://github.com/CPFL/Autoware/blob/master/ros/src/computing/perception/detection/fusion_tools/packages/range_vision_fusion/package.xml

Reply:
Removed unnecessary comment.